lib/autoupdate/start.rb

```diff
@@ -90,6 +90,14 @@ def start
         <string>#{log_std}</string>
         <key>StartInterval</key>
         <integer>#{interval}</integer>
+        <key>LowPriorityBackgroundIO</key>
+        <true/>
+        <key>LowPriorityIO</key>
+        <true/>
+        <key>Nice</key>
```

Comment:
I think we might not need a `Nice` value here?

From the `launchd.plist` manpage

<pre>
     ProcessType <string>
     This optional key describes, at a high level, the intended purpose of the job.
     The system will apply resource limits based on what kind of job it is. If left
     unspecified, the system will apply light resource limits to the job, throt-
     tling its CPU usage and I/O bandwidth. <b>This classification is preferable to
     using the HardResourceLimits, SoftResourceLimits and Nice keys.</b> The following
     are valid values:
</pre>

The manpage seems to suggest LowPriorityBackgroundIO and LowPriorityIO are independent of Nice/ProcessType and we should still set them.

Reply:
@danielcompton's comment seems like a reasonable adjustment here.